### PR TITLE
chore(deps): updated general dependencies

### PR DESCRIPTION
**Pull request by cristiam86**
Fixes N/A

# What

- Grouped renovate bot dependencies upgrades

# Why

- Maintain project updated

# Testing done

- Tested the studio

# Decisions made

# Checks

- [X] I have tested this code
- [X] I have reviewed my own PR
- [ ] I have created an issue for this PR
- [X] I have set a descriptive PR title compliant with [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)

# Reviewing tips

Execute tests, test Studio frontend

# User facing release notes

Dependencies across the whole project have been upgraded.

### CODE REVIEW

**Comment by cristiam86** [comment]
@AgustinRamiroDiaz unit, integration and e2e tests pass in local. Here they fail because there is no space left on device: https://github.com/yeagerai/genlayer-studio/actions/runs/12265932777/job/34244927585#step:7:2917

I'll make an issue to organize better the CI

File: webrequest/requirements.txt [hunk 1]
```diff
@@ -1,7 +1,9 @@
 Flask-JSONRPC==3.0.1
-Flask[async]==3.0.3
+Flask[async]==3.1.0
 python-dotenv==1.0.1
-pytest==8.3.3
+beautifulsoup4==4.12.3
+selenium==4.26.1
+webdriver-manager==4.0.2
```
Comment (AgustinRamiroDiaz) [inline webrequest/requirements.txt:6]: there are new packages here that I think shouldn't be

File: backend/protocol_rpc/requirements.txt [hunk 2]
```diff
@@ -11,12 +11,15 @@ numpy==1.26.4
...
-eth-utils==5.0.0
+debugpy==1.8.8
+aiohttp==3.11.0
+openai==1.54.4
+anthropic==0.39.0
+SQLAlchemy[asyncio]==2.0.36
+alembic==1.14.0
+eth-account==0.13.4
+eth-utils==5.1.0
+sentence-transformers==3.3.0
```
[the remark this answers is not in the document]
Comment (cristiam86) [reply backend/protocol_rpc/requirements.txt]: Removed, maybe I trusted too much in the bot :D

File: webrequest/requirements.txt (hunk 1 above)
Comment (cristiam86) [reply webrequest/requirements.txt]: Removed, maybe I trusted too much in the bot :D

**Review by AgustinRamiroDiaz — APPROVED** [review]
@cristiam86 the tests should pass now (the "no space" error was probably due to `sentence-transformers`)

And the bot probably added those dependencies due to having old data, because it added packages that we used at some point

**Pull request merged**